docs/t-sql/functions/char-transact-sql.md

```diff
@@ -43,10 +43,10 @@ CHAR ( integer_expression )
   
 ## Arguments  
 *integer_expression*  
-An integer from 0 through 255. `CHAR` returns a `NULL` value for integer expressions outside this range, or when then integer expresses only the first byte of a double-byte character.
+An integer from 0 through 255. `CHAR` returns a `NULL` value for integer expressions outside this range, or when no one character matches exactly the integer by both codepoint value and encoded value. Many common character sets share ASCII as a sub-set and will return the same character for integer values in the range 0 through 127.

```

Comment:
While correct, it's a more convoluted explanation that IMO is less clear to the average user. Please keep the "when then integer expresses only the first byte of a double-byte character." but also keep your edit "Many common character sets share ASCII as a sub-set and will return the same character for integer values in the range 0 through 127."

Reply:
I agree with your objection, but I still think that "first byte of a double-byte" character language is both misleading and incorrect. You'd get NULL when asking for the first, second, third, or fourth bytes of a multibyte UTF-8 character. How about "or not associated with a complete character compatible with the return type"?